openmls/src/group/core_group/mod.rs

```diff
@@ -640,7 +640,7 @@ impl CoreGroup {
         if epoch < self.context().epoch() {
             self.message_secrets_store
                 .secrets_for_epoch_mut(epoch)
-                .ok_or_else(|| {
+                .ok_or({
```

Comment:
Why? `ok_or_else` is preferable for this use case.

Reply:
That was clippy warning and I accepted the fix. I didn't check the previous PR for clippy warnings, but I guess it originated there.

Reply:
Here's the lint: https://rust-lang.github.io/rust-clippy/master/index.html#unnecessary_lazy_evaluations